### ISSUE TITLE
SimpleApp1 integration test doesn't pass on win32 jre 1.5.0_07

### ISSUE THREAD

**Issue by GoogleCodeExporter** [ankit4u3/jtracert#12]
```
What steps will reproduce the problem?
1. Install 1.5.0_07-b03 Sun JVM on 32 bit Windows XP
2. Try to execute integration tests: ant integration.test

What is the expected output? What do you see instead?

All tests should pass. Instead we get following error from integration test
SampleApp1:

Testcase: testSimpleApp1 took 0.672 sec
    Caused an ERROR
null
java.util.NoSuchElementException
    at java.util.LinkedList.remove(LinkedList.java:644)
    at java.util.LinkedList.removeFirst(LinkedList.java:134)
    at java.util.LinkedList.remove(LinkedList.java:485)
    at
com.google.code.jtracert.samples.JTracertSerializableTcpServer.getMethodCall(Unk
nown
Source)
    at com.google.code.jtracert.samples.SimpleApp1Test.testSimpleApp1(Unknown
Source)


```

Original issue reported on code.google.com by `Dmitry.Bedrin` on 24 Feb 2009 at 7:49

**Comment by GoogleCodeExporter** [comment]
Original comment by `Dmitry.Bedrin` on 24 Feb 2009 at 7:50
- Added labels: **Priority-Critical**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in SVN revision r172
```

Original comment by `Dmitry.Bedrin` on 24 Feb 2009 at 9:58
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in jTracert version 0.1.0
```

Original comment by `Dmitry.Bedrin` on 17 Mar 2009 at 10:12
